### PR TITLE
Use telegram tag for mention

### PR DESCRIPTION
**Pull request by calmmage**

### CODE REVIEW

File: source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/send_discord_post_to_telegram/send_discord_post_to_telegram.py [hunk 1]
```diff
@@ -66,21 +69,42 @@ async def send_discord_post_to_telegram(
...
-        message.content = message.content.replace(f"<@{user_id}>", user.nick or user.display_name)
+        telegram_username = (
+            discord_alias_to_telegram_username.get(user.nick)
+            or discord_alias_to_telegram_username.get(user.global_name)
+            or discord_alias_to_telegram_username.get(user.name)
+        )
+
+        # - Replace <@913095424225706005> with <@telegram_username> or <name>
+
+        message.content = message.content.replace(
```
Comment (calmmage) [inline source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/send_discord_post_to_telegram/send_discord_post_to_telegram.py:104]: Umm. Why'd you removed the if here? What if the username is not yet in the mapping?

File: source/python/services/discord-to-telegram-forwarder/run_test.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+cd ${0%/*} && poetry run python discord_to_telegram_forwarder/main.py --env test
```
Comment (calmmage) [inline source/python/services/discord-to-telegram-forwarder/run_test.sh:1]: P.s. why no line ending?

File: git_secret/import_keys.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# add my_public_key.asc and my_private_key.gpg to this directory
+gpg --import my_public_key.asc
```
[the remark this answers is not in the document]
Comment (marklidenberg) [reply git_secret/import_keys.sh:2]: It's for git secret. It allows you to store encrypted files in git repo and decrypt them locally. 

It's ok, but tooling needed for convenient usage

File: source/python/libs/lessmore.utils/lessmore/utils/easy_printing/print_json.py [hunk 4]
```diff
@@ -4,7 +4,7 @@
 def print_json(value, default=str, copy: bool = False, **kwargs):
     # - Dump
 
-    s = json.dumps(value, default=default, **kwargs)
+    s = json.dumps(value, default=default, indent=1, **kwargs)
```
[the remark this answers is not in the document]
Comment (marklidenberg) [reply source/python/libs/lessmore.utils/lessmore/utils/easy_printing/print_json.py]: yeap

File: source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/run.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+from discord_to_telegram_forwarder.send_discord_post_to_telegram.send_discord_post_to_telegram import (
+    send_discord_post_to_telegram,
+)
+from discord_to_telegram_forwarder.update_comments_counter.update_comments_counter import update_comments_counter
+from pymaybe import maybe
+
+from lessmore.utils.configure_loguru.configure_loguru import configure_loguru
+
+
+async def run(env: Literal["test", "prod"]):
```
[the remark this answers is not in the document]
Comment (marklidenberg) [reply source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/run.py:18]: Твой вариант лучше

File: source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/send_discord_post_to_telegram/send_discord_post_to_telegram.py (hunk 1 above)
Comment (marklidenberg) [reply source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/send_discord_post_to_telegram/send_discord_post_to_telegram.py:104]: It's the same code, I just put if inside

File: source/python/services/discord-to-telegram-forwarder/run_test.sh (hunk 2 above)
Comment (marklidenberg) [reply source/python/services/discord-to-telegram-forwarder/run_test.sh:1]: hmm

File: source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/main.py [hunk 6]
```diff
@@ -1,61 +1,13 @@
...
-    async with client:
-        await client.start(token=deps.config.discord_token)
+def main(env: Literal["test", "prod"]):
+    run(env=env)
 
 
 if __name__ == "__main__":
-    configure_loguru()
-    asyncio.run(main())
+    fire.Fire(main)
```
[the remark this answers is not in the document]
Comment (marklidenberg) [reply source/python/services/discord-to-telegram-forwarder/discord_to_telegram_forwarder/main.py]: Typer is better, fire is already installed :)

**Pull request merged**